docs/helm.md

```diff
@@ -8,15 +8,52 @@ Tilt supports Helm out-of-the-box.
 
 There are two ways you can use Helm charts with Tilt.
 
-## Basic Helm
+- Install a remote Helm chart from the rich library of existing charts.
 
-If you have a local Helm chart and want to deploy it as-is, add a `helm()` call to your `Tiltfile`:
+- Iterate on a local Helm chart that you're building.
 
-```python
-k8s_yaml(helm('path/to/chart/dir'))
+Let's dig into each one.
+
+## Remote charts
+
+The `helm_remote` extension lets you load from a library of existing charts.
```

Comment:
it might be worth giving some context on "a library of existing charts":
1. maybe call it "helm repository" vs "library", to give a better term to google for and more confidence that's what we're referring to
2. link to an example repository or list of repositories?
3. `helm repo` stuff? (e.g. https://github.com/tilt-dev/tilt-extensions/issues/157)

Reply:
added a link to Artifact hub